### PR TITLE
Fix `old_name` field in `AuditEntry`

### PR DESCRIPTION
**Pull request by lmartinking**
Much to my chagrin, the audit log view in the site manager does not follow the shape of the API response.

I previously raised a PR #2843 which was merged yesterday. Unfortunately, it does not work with the payload returned from the API.

My sincere apologies about this.

### CODE REVIEW

File: github/orgs_audit_log.go [hunk 1]
```diff
@@ -122,6 +121,13 @@ type AuditEntry struct {
 	Visibility             *string                 `json:"visibility,omitempty"` // The repository visibility, for example `public` or `private`.
 	WorkflowID             *int64                  `json:"workflow_id,omitempty"`
 	WorkflowRunID          *int64                  `json:"workflow_run_id,omitempty"`
+
+	AuditEntryData `json:"data,omitempty"`
```
Comment (gmlewis) [inline github/orgs_audit_log.go:125]:
```suggestion
	Data  *AuditEntryData `json:"data,omitempty"`
```

File: github/orgs_audit_log.go [hunk 2]
```diff
@@ -122,6 +121,13 @@ type AuditEntry struct {
...
+}
+
+// Some audit entries have additional information stuffed into a `data` field.
```
Comment (gmlewis) [inline github/orgs_audit_log.go:128]:
```suggestion
// AuditEntryData represents additional information stuffed into a `data` field.
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @lmartinking !
Now worries.  I'm glad you caught it before I cut the new release.  😄

**Comment by lmartinking** [comment]
@gmlewis I have one more field to add, which I will do in a other PR

**Comment by lmartinking** [comment]
@gmlewis do you want the structs to mirror the shape of the API or would you prefer it flattened? Which was why I had it as a composite member.

**Comment by gmlewis** [comment]
> @gmlewis do you want the structs to mirror the shape of the API or would you prefer it flattened? Which was why I had it as a composite member.

We typically do _not_ use composite members in this repo.

**Review by gmlewis — APPROVED** [review]
Thank you, @lmartinking !
LGTM.
Merging.

**Pull request merged**